sympy/assumptions/ask.py

```diff
@@ -1484,13 +1484,16 @@ def get_known_facts():
         Equivalent(Q.prime, Q.integer & Q.positive & ~Q.composite),
         Implies(Q.integer, Q.rational),
         Implies(Q.rational, Q.algebraic),
+        Implies(Q.irrational, Q.finite),
```

Comment:
Is this redundant because of `Equivalent(Q.irrational | Q.rational, Q.real & Q.finite)`?

Reply:
Good point. I removed it and ran ask_update.py and nothing changed.